lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -377,18 +379,18 @@ namespace Js
             // clear out all inline caches to remove our proto inline caches from the thread context
             threadContext->ClearInlineCaches();
 
+            ClearInlineCaches();
```

Comment:
The loop in threadContext->ClearInlineCaches will loop through and call scriptContext->ClearInlineCaches; I'm just moving the particular call out as we have removed itself from the linklist.